src/components/EditImageModal/index.jsx

```diff
@@ -551,6 +582,24 @@ export default class EditImageModal extends React.Component {
     return feedback;
   }
 
+  validateImageDimensions = () => {
+    let feedback = { isValid: true };
+    const { height, width } = this.state.imageDimensions;
+
+    if (((height !== '' && height !== undefined) && (!Number.isInteger(height) || height <= 0)) ||
+      ((width !== '' && width !== undefined) && (!Number.isInteger(width) || width <= 0))) {
```

Comment:
This if statement is really hard to read -- I thought we were only checking if these values were negative? Can we make this more readable?

Reply:
Because we allow the fields to be empty, I need to check if they are `''` or `undefined` which complicates things. I'm going to abstract this condition to a separate function, what do you think of this?

```javascript
  isPositiveIntegerOrEmpty = (val) => {
    if (val === '' || val === undefined) {
      return true;
    }
    if (Number.isInteger(val) && val > 0) {
      return true;
    }
    return false;
  }
```

Reply:
👍

Reply:
Wayy easier to read :) Thanks!